src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/Worker.java

```diff
@@ -659,6 +701,7 @@ private void initialize() {
                 LOG.error("Caught exception when initializing LeaseCoordinator", e);
                 lastException = e;
             } catch (Exception e) {
+                LOG.error("Caught exception when initializing worker", e);
```

Comment:
Let's not change the logging behavior here. This will keep on spitting errors for all retry attempts. Could you remove this change.

Reply:
Removed.